txparser.go

```diff
@@ -3,16 +3,16 @@ package dcrlibwallet
 import (
 	"fmt"
 	"github.com/decred/dcrd/chaincfg/chainhash"
-	"github.com/decred/dcrwallet/wallet"
+	w "github.com/decred/dcrwallet/wallet/v3"
 )
 
-func (lw *LibWallet) decodeTransactionWithTxSummary(txSummary *wallet.TransactionSummary,
+func (wallet *Wallet) decodeTransactionWithTxSummary(txSummary *w.TransactionSummary,
 	blockHash *chainhash.Hash) (*Transaction, error) {
 
-	var blockHeight int32 = -1
+	var blockHeight int32 = BlockHeightInvalid
```

Comment:
This `BlockHeightInvalid` constant can be declared in this file instead of elsewhere.